src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,9 @@
+package core.basesyntax;
+
+public class ColorSupplier {
+    private static final String[] COLORS = {"RED", "ORANGE", "GREEN", "GRAY", "YELLOW"};
+
+    public static String getRandomColor() {
```

Comment:
```suggestion
    public String getRandomColor() {
```
please don't use static methods - they are in general bad practice